dbt/adapters/sqlserver/sql_server_adapter.py

```diff
@@ -3,7 +3,8 @@
 
 import dbt.exceptions
 from dbt.adapters.fabric import FabricAdapter
-from dbt.contracts.graph.nodes import ConstraintType, ModelLevelConstraint
+from dbt.contracts.graph.nodes import ConstraintType
+from dbt_common.contracts.constraints import ModelLevelConstraint
```

Comment:
you should not use this 
from dbt.contracts.graph.nodes import ConstraintType
and instead:
from dbt_common.contracts.constraints import ConstraintType, ModelLevelConstraint